red.py

```diff
@@ -154,6 +154,9 @@ def user_allowed(self, message):
         else:
             return True
 
+    def prefix_manager(self, bot, msg):
```

Comment:
why does this require a `bot` arg? It neither uses nor needs it since `self` is a Bot instance